pkg/scheduler/framework/v1alpha1/types.go

```diff
@@ -150,12 +148,18 @@ func NewPodInfo(pod *v1.Pod) *PodInfo {
 		}
 	}
 
+	requiredAffinityTerms, podErr := getAffinityTerms(pod, schedutil.GetPodAffinityTerms(pod.Spec.Affinity))
+	requiredAntiAffinityTerms, podErr := getAffinityTerms(pod, schedutil.GetPodAntiAffinityTerms(pod.Spec.Affinity))
+	weightAffinityTerms, podErr := getWeightedAffinityTerms(pod, preferredAffinityTerms)
```

Comment:
worth wrapping (`fmt.Errorf`) the errors here to indicate which one failed

Reply:
Done, I wrapped them in this spot with just `requiredAffinityTerm,preferredAntiAffinity,etc`. The other option would be wrapping the error in `newAffinityTerm` with the labelSelector that failed but I don't have a preference